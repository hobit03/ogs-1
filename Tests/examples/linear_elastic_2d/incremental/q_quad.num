GeoSys-NUM: Numerical Parameter ----------------------------------------
#NUMERICS
 $PCS_TYPE
  DEFORMATION
 $LINEAR_SOLVER 
; method error_tolerance max_iterations theta precond storage
  5      1 1.e-10       1000           1.0   1     2
;  2      1 1.e-10       1000           1.0   1     2
 $NON_LINEAR_SOLVER 
; method error_tolerance max_iterations relaxation
;  LINEAR 1e-10            1            0.0
;  PICARD 1e-10            2            0.0
  NEWTON 1e-6 1e-10    3            0.0
#STOP

