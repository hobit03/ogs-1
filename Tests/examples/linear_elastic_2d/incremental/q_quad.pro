<?xml version="1.0"?>
<ogs6>
	<coupling>
		<P algorithm="Serial" convergence="FemFunctionConvergenceCheck" max_itr="1" epsilon="1e-4">
			<out>DISPLACEMENT</out>
			<out>gp_strain</out>
			<out>gp_stress</out>
			<out>STRAIN</out>
			<out>STRESS</out>
			<problems>
				<M type="INCREMENTAL_DEFORMATION" name="DEFORMATION">
					<out>DISPLACEMENT</out>
					<out>gp_strain</out>
					<out>gp_stress</out>
				</M>
				<M type="NODAL_STRESS_STRAIN">
					<in>gp_strain</in>
					<in>gp_stress</in>
					<out>STRAIN</out>
					<out>STRESS</out>
				</M>
			</problems>
		</P>
	</coupling>
</ogs6>

