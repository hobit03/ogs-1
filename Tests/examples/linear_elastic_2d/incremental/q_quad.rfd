;T
#CURVE
0.0	0
864000	-1e+7
#STOP


