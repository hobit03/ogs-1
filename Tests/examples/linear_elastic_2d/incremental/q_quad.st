GeoSys-ST: Source Terms ------------------------------------------------
#SOURCE_TERM
 $PCS_TYPE
  DEFORMATION
 $PRIMARY_VARIABLE
  DISPLACEMENT_Y1
 $GEO_TYPE
  POLYLINE TOP
 $DIS_TYPE   
  CONSTANT_NEUMANN 1
;  CONSTANT_NEUMANN -1e+6
 $TIM_TYPE
  CURVE 1
#STOP


