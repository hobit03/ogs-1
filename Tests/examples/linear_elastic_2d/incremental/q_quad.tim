GeoSys-OUT: Output ------------------------------------------------
#TIME_STEPPING
 $PCS_TYPE
  DEFORMATION
 $TIME_STEPS
  10  86400
;  30  60
 $TIME_END
  1e+99
 $TIME_START
  0.0
#STOP

