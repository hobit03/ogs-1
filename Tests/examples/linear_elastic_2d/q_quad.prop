<?xml version="1.0"?>
<coupling>
	<P algorithm="Serial" convergence="FemFunctionConvergenceCheck" max_itr="1" epsilon="1e-4">
		<out>u</out>
		<out>gp_strain</out>
		<out>gp_stress</out>
		<out>nod_strain</out>
		<out>nod_stress</out>
		<problems>
			<M name="DEFORMATION">
				<out>u</out>
			</M>
			<M name="ELEMENT_STRESS_STRAIN">
				<in>u</in>
				<out>gp_strain</out>
				<out>gp_stress</out>
			</M>
			<M name="NODAL_STRESS_STRAIN">
				<in>gp_strain</in>
				<in>gp_stress</in>
				<out>nod_strain</out>
				<out>nod_stress</out>
			</M>
		</problems>
	</P>
</coupling>

