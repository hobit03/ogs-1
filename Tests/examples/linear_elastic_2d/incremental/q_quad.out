GeoSys-OUT: Output ------------------------------------------------
#OUTPUT
 $NOD_VALUES
  DISPLACEMENT
  DISPLACEMENT_X1
  DISPLACEMENT_Y1
  STRESS_XX
  STRESS_XY
  STRESS_YY
  STRESS_ZZ
  STRESS_XZ
  STRESS_YZ
  STRAIN_XX
  STRAIN_XY
  STRAIN_YY
 $ELE_VALUES
  gp_stress_XX
  gp_stress_XY
  gp_stress_YY
  gp_stress_ZZ
  gp_stress_XZ
  gp_stress_YZ
  gp_strain_XX
  gp_strain_XY
  gp_strain_YY
 $GEO_TYPE
  DOMAIN
 $DAT_TYPE
  PVD
 $TIM_TYPE
  STEPS 1
#STOP


